apps/controlled-drugs/fields/index.js

```diff
@@ -0,0 +1,93 @@
+module.exports = {
+  'company-name': {
+    mixin: 'input-text',
+    validate: ['required', 'notUrl', { type: 'minlength', arguments: 2 }, { type: 'maxlength', arguments: 200 }],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'company-number': {
+    mixin: 'input-text',
+    validate: [], // additional validation rules added in custom-validation.js
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  telephone: {
+    mixin: 'input-text',
+    validate: ['required'], // additional validation covered in custom-validation.js
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  email: {
+    mixin: 'input-text',
+    validate: ['required', 'email'],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'website-url': {
+    mixin: 'input-text',
+    validate: ['url', { type: 'maxlength', arguments: 250 }],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'licence-holder-address-line-1': {
+    mixin: 'input-text',
+    validate: ['required', 'notUrl', { type: 'maxlength', arguments: 250 }],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'licence-holder-address-line-2': {
+    mixin: 'input-text',
+    validate: ['notUrl', { type: 'maxlength', arguments: 250 }],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'licence-holder-town-or-city': {
+    mixin: 'input-text',
+    validate: ['required', 'notUrl', { type: 'maxlength', arguments: 250 }],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'licence-holder-postcode': {
+    mixin: 'input-text',
+    validate: ['required', 'postcode'],
+    formatter: ['ukPostcode'],
+    className: ['govuk-input', 'govuk-input--width-10']
+  },
+  'is-premises-address-same': {
+    isPageHeading: 'true',
+    mixin: 'radio-group',
+    validate: ['required'],
+    className: ['govuk-radios govuk-radios--inline'],
+    options: [
+      {
+        value: 'yes'
+      },
+      {
+        value: 'no'
+      }
+    ]
+  },
+  'premises-address-line-1': {
+    validate: ['required', { type: 'maxlength', arguments: [250]}, 'notUrl'],
+    mixin: 'input-text',
```

Comment:
minor note, I would suggest for all `premises-address-` fields to move `mixin` above `validate` for readability